prisma-fmt/tests/code_actions/scenarios/create_missing_block_composite_type_crlf/schema.prisma

```diff
@@ -1,13 +1,13 @@
-generator client {
-    provider = "prisma-client-js"
-}
-
-datasource db {
-    provider = "mongodb"
-    url      = env("DATABASE_URL")
-}
-
-type Kattbjorn {
-    name   String
-    friend Animal
-}
+generator client {
```

Comment:
oops, I think `jj` ignored the `.gitattributes` and converted crlf to lf on my branch